_config.yml

```diff
@@ -20,3 +20,5 @@ languages:
     code: "tr"
   - name: "Russian"
     code: "ru"
+  - name: "Chinese"
+    code: "ch"
```

Comment:
I added Traditional Chinese as zh so it's possibly ok to leave this as ch. Or we can fix it so they are differentiated as `zh_CN` and `zh_TW`?

Reply:
Yes, you can submit a new PR for revision